src/instructlab/eval/mmlu.py

```diff
@@ -206,20 +210,29 @@ def run(self) -> tuple:
             individual_scores   Individual MMLUBranch scores for each task in the task group
         """
         logger.debug(locals())
+
+        if not os.path.exists(self.sdg_path):
+            raise SDGPathNotFoundError(self.sdg_path)
```

Comment:
Hard to get this stuff perfect, but worth seeing if you're ok with what happens if the file isn't readable (in terms of permissions)

Reply:
For the directory itself, it's already checked above: Error: Invalid value for '--sdg-path': Path 'tests/testdata/mmlu_branch' is not readable.

But I'll go ahead and add a check here since the library is generic.

Reply:
Fixed, thanks for the suggestions!